spec/models/comment_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { Comment.new(text: 'Hello World') }
+  before { subject.save }
+
+  it 'comment_counter method should raise error without post' do
+    expect { subject.comment_counter }.to raise_error(NoMethodError)
+  end
+end
```

Comment:
- [ ] Let us endeavor to be thorough in our specs. Kindly add positive scenarios where the comment save works and the comments counter of the related post is updated in stead of only just raising errors.

Reply:
Hi @julie-ify .... Thanks for the review but I would like you to assist in resolving most of the feedback you have given. I have a bit of difficulty understanding most of the feedback and even how to go around them. THANKS